src/coreclr/src/tools/aot/jitinterface/jitinterface.h

```diff
@@ -191,1474 +195,1660 @@ class JitInterfaceWrapper
-    {
-        CorInfoException* pException = nullptr;
-        _callbacks->reportTailCallDecision(_thisHandle, &pException, callerHnd, calleeHnd, fIsTailPrefix, tailCallResult, reason);
-        if (pException != nullptr)
-            throw pException;
-    }
-
-    virtual void getEHinfo(void* ftn, unsigned EHnumber, void* clause)
-    {
-        CorInfoException* pException = nullptr;
-        _callbacks->getEHinfo(_thisHandle, &pException, ftn, EHnumber, clause);
-        if (pException != nullptr)
-            throw pException;
-    }
-
-    virtual void* getMethodClass(void* method)
-    {
-        CorInfoException* pException = nullptr;
-        void* _ret = _callbacks->getMethodClass(_thisHandle, &pException, method);
-        if (pException != nullptr)
-            throw pException;
-        return _ret;
-    }
-
-    virtual void* getMethodModule(void* method)
-    {
-        CorInfoException* pException = nullptr;
-        void* _ret = _callbacks->getMethodModule(_thisHandle, &pException, method);
-        if (pException != nullptr)
-            throw pException;
-        return _ret;
-    }
-
-    virtual void getMethodVTableOffset(void* method, unsigned* offsetOfIndirection, unsigned* offsetAfterIndirection, bool* isRelative)
-    {
-        CorInfoException* pException = nullptr;
-        _callbacks->getMethodVTableOffset(_thisHandle, &pException, method, offsetOfIndirection, offsetAfterIndirection, isRelative);
-        if (pException != nullptr)
-            throw pException;
-    }
-
-    virtual void* resolveVirtualMethod(void* virtualMethod, void* implementingClass, void* ownerType)
-    {
-        CorInfoException* pException = nullptr;
-        void* _ret = _callbacks->resolveVirtualMethod(_thisHandle, &pException, virtualMethod, implementingClass, ownerType);
-        if (pException != nullptr)
-            throw pException;
-        return _ret;
-    }
 
-    virtual void* getUnboxedEntry(void* ftn, bool* requiresInstMethodTableArg)
-    {
-        CorInfoException* pException = nullptr;
-        void* _ret = _callbacks->getUnboxedEntry(_thisHandle, &pException, ftn, requiresInstMethodTableArg);
-        if (pException != nullptr)
-            throw pException;
-        return _ret;
-    }
-
-    virtual void* getDefaultEqualityComparerClass(void* elemType)
-    {
-        CorInfoException* pException = nullptr;
-        void* _ret = _callbacks->getDefaultEqualityComparerClass(_thisHandle, &pException, elemType);
-        if (pException != nullptr)
-            throw pException;
-        return _ret;
-    }
-
-    virtual void expandRawHandleIntrinsic(void* pResolvedToken, void* pResult)
-    {
-        CorInfoException* pException = nullptr;
-        _callbacks->expandRawHandleIntrinsic(_thisHandle, &pException, pResolvedToken, pResult);
-        if (pException != nullptr)
-            throw pException;
-    }
+    virtual unsigned int getMethodAttribs(
+          void* ftn)
+{
```

Comment:
Nit: The indentation is off.

Reply:
The generator is ugly enough as it is. Unless there is pushback, I'd prefer to leave the indentation.